guides/common/attributes-orcharhino.adoc

```diff
@@ -52,6 +52,7 @@
 :atix-kb-register-to-occ: https://atixservice.zendesk.com/hc/de/articles/9464966748444[Registering {ProjectServer} to OCC]
 :atix-kb-tech-previews: https://atixservice.zendesk.com/hc/de/articles/16772654610844[Technical Previews]
 // Overwritten in downstream for docs.orcharhino.com
+:client-errata-id-explanation: numerical erratum ID provided by Foreman/Katello.
```

Comment:
```suggestion
:client-errata-id-explanation: numerical erratum ID provided by Foreman/Katello
```
It would result in two periods at the end of sentences.

Apropos, I can see it used only once, but I'm assuming you will re-use it in future.

Reply:
Thx, applied.

I will overwrite it for different Client OS in downstream.